contrib/gorm.io/gorm.v1/gorm_test.go

```diff
@@ -343,3 +343,53 @@ func TestContext(t *testing.T) {
 		assert.Equal(t, testCtx.Value(key(contextKey)), ctx.Value(key(contextKey)))
 	})
 }
+
+func TestError(t *testing.T) {
+	mt := mocktracer.Start()
+	defer mt.Stop()
+
+	assertErrCheck := func(t *testing.T, mt mocktracer.Tracer, errExist bool, opts ...Option) {
+		sqltrace.Register("pgx", &stdlib.Driver{})
+		sqlDb, err := sqltrace.Open("pgx", pgConnString)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		db, err := Open(postgres.New(postgres.Config{Conn: sqlDb}), &gorm.Config{})
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		db.AutoMigrate(&Product{})
+
+		db = db.WithContext(context.Background())
```

Comment:
Is this line necessary? If so, why?

Reply:
I just followed the code already done in https://github.com/DataDog/dd-trace-go/blob/v1/contrib/gopkg.in/jinzhu/gorm.v1/gorm_test.go#L361 and I adapted here.
I'll check if this line is necessary.

Reply:
I removed this line.